extensions/amp-analytics/0.1/iframe-transport-vendors.js

```diff
@@ -25,6 +25,6 @@
  * @const {!JsonObject}
  */
 export const IFRAME_TRANSPORTS = /** @type {!JsonObject} */ ({
-  'bg': 'https://tpc.googlesyndication.com/b4a/b4a-runner.html',
+  'bg': 'https://tpc.googlesyndication.com/b4a/experimental/b4a-runner.html',
```

Comment:
as discussed offline, could you add it as a new config block

```js
export const IFRAME_TRANSPORTS_CANARY = {
  'bg': 'https://tpc.googlesyndication.com/b4a/experimental/b4a-runner.html',
  'moat': 'https://z.moatads.com/ampanalytics093284/iframe.html',
}
```

then you will change `vendors.js`

```js
mergeIframeTransportConfig(ANALYTICS_CONFIG, isCanary(self) ? IFRAME_TRANSPORTS_CANARY : IFRAME_TRANSPORTS);

```

Reply:
Done.